### PR TITLE
Station 1

### PR DESCRIPTION
**Pull request by Jiaqi-beep**
- NCL script: https://www.ncl.ucar.edu/Applications/Scripts/station_1.ncl
- NCL plot: https://www.ncl.ucar.edu/Applications/Images/station_1_lg.png
- Issue: #382 

station 1

### CODE REVIEW

**Comment by michaelavs** [comment]
I'm having issues with the colorbar displaying on this one, is anyone else having a similar issue? I've tried adjusting the figure size to be larger and also running the sphinx build, but all are not displaying the colorbar. Here is what I see for reference:
image

@anissa111 @erinlincoln

**Comment by erinlincoln** [comment]
It is showing differently on my end in terms of label size (not sure why), but I was able to get it to show by changing line 79 to:
cax = fig.add_axes([0.9, 0.12, 0.05, 0.75])
This is what it looks like on my end as a result:
Screen Shot 2021-07-06 at 3 50 49 PM

**Comment by Jiaqi-beep** [comment]
Thanks! I will update this. It seems that I keep having issues with elements not showing. I think my Spyder config displays all the plot elements even when they are actually cut off because of the limited figsize. @erinlincoln Do you happen to know how to change the spyder plot configuration without optimizing the plot presentation?

File: Plots/Station/NCL_station_1.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                   cax=cax,
+                   ticks=flevels[::2],
+                   drawedges=False,
+                   extendrect=True)
+
+# Set colorbar ticklabel font size
+cab.ax.yaxis.set_tick_params(length=0, labelsize=20)
+
+# Show the plot
+plt.savefig('hello.png')
```
Comment (michaelavs) [inline Plots/Station/NCL_station_1.py:130]:
```suggestion
```
I think this may have been left over

**Review by michaelavs — COMMENTED** [review]
Looks great overall! Just some minor formatting changes!

**Comment by Jiaqi-beep** [comment]
Thank you! @michaelavs

File: Plots/Station/NCL_station_1.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+pwv_lat1d = ds.LAT
+pwv_lon1d = ds.LON
+
+###################################################
+# Plot
+
+# Generate figure (set its size (width, height) in inches)
+fig = plt.figure(figsize=(12, 10))
+
+# Generste axes
```
Comment (anissa111) [inline Plots/Station/NCL_station_1.py:46]:
```suggestion
# Generate axes
```

File: Plots/Station/NCL_station_1.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+ax = plt.axes(projection=ccrs.PlateCarree())
+
+# Specify contour and contourf levels
+clevels = np.arange(25, 51, 5)
+flevels = np.arange(16, 51, 1)
+
+# Specify colormap and reverse it
```
Comment (anissa111) [inline Plots/Station/NCL_station_1.py:53]:
```suggestion
# Specify colormap
```
The reversal doesn't actually happen here, I thought this comment might be a little confusing

File: Plots/Station/NCL_station_1.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+                       alpha=0.85,
+                       levels=flevels,
+                       antialiased=True,
+                       zorder=3)
+
+# Add coordinate markers on the plot
+ax.plot(pwv_lon1d, pwv_lat1d, marker='o', linewidth=0, color='black', zorder=4)
+
+# Add state boundaries other lake features
+ax.add_feature(cfeature.STATES, edgecolor='silver', linestyle=(0, (5, 10)), zorder=2)
```
Comment (anissa111) [inline Plots/Station/NCL_station_1.py:82]:
```suggestion
ax.add_feature(cfeature.STATES, edgecolor='gray', linestyle=(0, (5, 10)), zorder=2)
```

I think the borders should be a little darker here. On my side, it's kind of hard to see them under the contours. This change makes them just a little darker. Feel free to disagree with me on this, though!

**Review by anissa111 — COMMENTED** [review]
This looks good so far! I've left a couple minor changes below.

File: Plots/Station/NCL_station_1.py (hunk 4 above)
Comment (Jiaqi-beep) [reply Plots/Station/NCL_station_1.py:82]: I agree with you on this. Thanks!

**Review by michaelavs — APPROVED** [review]
Thanks for making those changes!

**Review by anissa111 — APPROVED** [review]
This looks good! I fixed a typo, but when the tests go through, this is okay to merge!

**Pull request merged**